description.txt

```diff
@@ -1 +1,19 @@
-This is a description file for git training session, now resolve conflicts 
\ No newline at end of file
+This is a description file for git training session, now resolve conflicts
```

Comment:
@akshay-mundra better commit message should be there.
docs: add git training session mom